src/components/widgets/PlayerNumberChart.tsx

```diff
@@ -0,0 +1,61 @@
+// @ts-nocheck
+
+import { onMount } from "solid-js"
+import { Chart, Title, Tooltip, Colors, TimeScale, type ChartOptions, type ChartData } from "chart.js"
+import { Bar } from "solid-chartjs"
+import "chartjs-adapter-luxon"
+
+type PlayerNumberChartProps = {
+  labels: string[]
+  data: number[]
+}
+
+const longNumberFormatter = new Intl.NumberFormat("en-us")
+
+export function PlayerNumberChart(props: PlayerNumberChartProps) {
```

Comment:
GameNumberChart and PlayerNumberChart are essentially the same component. I suggest:
- Abstract it to 'BarChart'.
- Move the creating of the labels, values and title to outside of this component into stats.astro

From a Visual perspective I personally prefer line graphs for time series evolution but bars are fine here too.